src/main/app/e2e/features/1-authentication.feature

```diff
@@ -1,6 +1,9 @@
 Feature: Login
 
+  Background: 
+    Given Zac is live
+
   Scenario: Bob wants to login to ZAC
-    Given "Bob" navigates to "http://zaakafhandelcomponent-zac-dev.westeurope.cloudapp.azure.com"
+    Given "Bob" navigates to "http://localhost:8080/"
```

Comment:
I would not use an actual link, but a name that can then be 'replaced' in the test code with the environment you want to test on; being it local, test or another location.